src/applications/vaos/utils/events.js

```diff
@@ -88,12 +88,7 @@ export function recordAppointmentDetailsNullStates(attributes, nullStates) {
 
   recordEvent({
     event: `${GA_PREFIX}-null-states`,
-    type:
```

Comment:
Reverting since this is no longer needed after we remove the colliding type field on requests.